<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">


   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>

    <widget class="QPushButton" name="pushButton_C">
        <property name="geometry">
        <rect>
        <x>20</x>
        <y>180</y>
        <width>60</width>
        <height>60</height>
        </rect>
        </property>
        <property name="text">
        <string>C</string>
        </property>

    </widget>

   <widget class="QPushButton" name="pushButton_%">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_.">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_/">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_*">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_-">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_+">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton_=">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>

    <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
        <rect>
        <x>20</x>
        <y>260</y>
        <width>60</width>
        <height>60</height>
        </rect>
        </property>
        <property name="text">
        <string>7</string>
        </property>
    </widget>

    <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
        <rect>
        <x>20</x>
        <y>340</y>
        <width>60</width>
        <height>60</height>
        </rect>
        </property>
        <property name="text">
        <string>4</string>
        </property>
    </widget>

    <widget class="QPushButton" name="pushButton_1">
        <property name="geometry">
        <rect>
        <x>20</x>
        <y>420</y>
        <width>60</width>
        <height>60</height>
        </rect>
        </property>
        <property name="text">
        <string>1</string>
        </property>
    </widget>

    <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
        <rect>
        <x>20</x>
        <y>100</y>
        <width>300</width>
        <height>60</height>
        </rect>
        </property>
        <property name="font">
        <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        </font>
        </property>
    </widget>

  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
